### ISSUE TITLE
NYTimes and Toronto Star have new paywalls

### ISSUE THREAD

**Issue by cpmiller51** [iamadamdev/bypass-paywalls-firefox#128]
It appears the NYTimes has intro'd a new paywall system. After the 1 free article, subsequent articles display the upper third of the content; the lower 2/3rds is then hidden by a request to subscribe.

The Toronto Star (thestar.com) appears to have selectively introduced a similar scheme. Up till now, there's been a full-screen subscribe request with a large X at the top right. Click the X and see the article. Now, some articles are like that but others adopt the same one-third content/two-thirds subscribe approach done by the NYTimes.

Examples of the Toronto Star's paywalls are below:

**The big X version**
https://www.thestar.com/politics/federal/2019/02/11/ethics-commissioner-to-launch-inquiry-against-prime-ministers-office-over-snc-lavalin-case.html

**The one-third/two-thirds cover up with no dismissal option**
https://www.thestar.com/news/gta/2019/02/11/environment-canada-issues-special-weather-statement-for-tuesday.html

Thanks for all your efforts!

**Comment by iamadamdev** [comment]
NYT still works fine for me (just loaded 10+ articles).

For thestar you can just click the X for most of the articles and still read it, the example you posted seems that some of the articles they are adding a hard paywall.

**Comment by cpmiller51** [comment]
In the process of testing just now, I turned off every add-on except uBlock and BP. I fired up NYTimes and <face palm> I suddenly noticed that somehow I'd turned off uBlock on the NYTimes site. I turned it back on and now NYTimes is working fine. So...thanks for forcing me to engage my brain.

That said, I then turned OFF BP but left uBlock on. I next closed down Firefox, reopened it, fired up the NYTimes, and then fired up 26 separate articles. Every one of 'em loaded fine. It appears that on the NYTimes, BP is superfluous if one has uBlock Origin installed (and activated).

Re: Toronto Star. Thanks for trying and oh well. It now follows the path of the Globe & Mail, which also has a mix of "soft" and "hard" paywalls.
